src/lib.rs

```diff
@@ -391,24 +391,28 @@ fn bytes_range(offset: u64) -> String {
     format!("bytes={}-", offset)
 }
 
+pub enum SyncParam {
+    Local { path: String },
+    Bucket { bucket: String, path: String },
+}
+
+impl std::fmt::Debug for SyncParam {
```

Comment:
Just use `#[derive(Debug)]` on the type, no need to implement this manually unless you're customizing something.